src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -488,9 +488,13 @@ private function getType(string $type, bool $isCollection, string $className = n
      */
     private function computeDoc(Documentation $documentation, \ArrayObject $definitions, \ArrayObject $paths) : array
     {
+        /** @var Symfony\Component\Routing\RequestContext $requestContext */
+        $requestContext = $this->urlGenerator->getContext();
```

Comment:
OK, anyway, I'll introduce a new interface matching the RequestContext (so, `getHost()` and `getSchene()`) and inject it here.

If that sounds wrong, please let me know. /cc @dunglas